builders/build/templates/custom-runtime.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Custom Runtime
+description: Any custom runtime can be adapted to be deployed through Tanssi, provided that specific modules are implemented, and the required configurations are set.
+---
+
+# Adapting an Existing Runtime {: #adapting-existing-runtime }
```

Comment:
Maybe here we can add a ## after introduction to state what things you can't have. Like:

```
## Minimum Requirements

Already existing Substrate runtimes need to at least have both [Cumulus](#adding-cumulus-support) and [Tanssi-specific modules](#adding-tanssi-support).

Nevertheless, teams might've already implemented certain modules that can collide with some functionalities related to Tanssi, for example, block production and block authority assignment.
```

And expand a bit on that you can't have things like Aura etc.